ansible/tests/test.bats

```diff
@@ -1,11 +1,14 @@
-source "${BATS_TEST_DIRNAME}/../plan.sh"
+@test "package directory for package ident ${TEST_PKG_IDENT} exists" {
```

Comment:
Same comment as other PR's, I don't think we need to check this.

Reply:
Removed, also fixed use of `--binlink`